boltons/dictutils.pyi

```diff
@@ -0,0 +1,118 @@
+try:
+    from collections.abc import ItemsView, KeysView, ValuesView
+except ImportError:
+    from collections import KeysView, ValuesView, ItemsView
+
+from collections.abc import Callable, Generator, Hashable, Iterable, Iterator
+from typing import Any, Never
+
+class OrderedMultiDict(dict):
+    def __init__(self, *args, **kwargs) -> None: ...
+    def add(self, k: Hashable, v: Any) -> None: ...
+    def addlist(self, k: Hashable, v: Any) -> None: ...
+    def get(self, k: Hashable, default: Any | None = ...) -> Any: ...
+    def getlist(self, k: Hashable, default=...) -> list[Any]: ...
+    def clear(self) -> None: ...
+    def setdefault(self, k: Hashable, default: Any = ...) -> Any: ...
+    def copy(self) -> OrderedMultiDict: ...
+    @classmethod
+    def fromkeys(
+        cls, keys: list[Any], default: Any | None = ...
+    ) -> OrderedMultiDict: ...
+    def update(self, E: dict[Any, Any] | Iterable, **F: dict[str, Any]) -> None: ...
+    def update_extend(
+        self, E: dict[Any, Any] | Iterable, **F: dict[str, Any]
+    ) -> None: ...
+    def __setitem__(self, k: Hashable, v: Any) -> None: ...
+    def __getitem__(self, k: Hashable) -> Any: ...
+    def __delitem__(self, k: Hashable) -> None: ...
+    def __eq__(self, other: OrderedMultiDict) -> bool: ...
+    def __ne__(self, other: OrderedMultiDict) -> bool: ...
+    def pop(self, k: Hashable, default: Any = ...) -> None: ...
+    def popall(self, k: Hashable, default: Any = ...) -> None: ...
+    def poplast(self, k: Hashable = ..., default: Any = ...) -> None: ...
+    def iteritems(self, multi: bool = ...) -> Generator[Any, None, None]: ...
+    def iterkeys(self, multi: bool = ...) -> Generator[Any, None, None]: ...
+    def itervalues(self, multi: bool = ...) -> Generator[Any, None, None]: ...
+    def todict(self, multi: bool = ...) -> dict[Hashable, Any]: ...
+    def sorted(
+        self, key: Any | None = ..., reverse: bool = ...
+    ) -> OrderedMultiDict: ...
+    def sortedvalues(
+        self, key: Any | None = ..., reverse: bool = ...
+    ) -> OrderedMultiDict: ...
+    def inverted(self) -> OrderedMultiDict: ...
+    def counts(self) -> OrderedMultiDict: ...
+    def keys(self, multi: bool = ...) -> list[Hashable]: ...
+    def values(self, multi: bool = ...) -> list[Any]: ...
+    def items(self, multi: bool = ...) -> list[Generator[Any, None, None]]: ...
+    def __iter__(self) -> Generator[Any, None, None]: ...
+    def __reversed__(self) -> Generator[Any, None, None]: ...
+    def viewkeys(self) -> KeysView: ...
+    def viewvalues(self) -> ValuesView: ...
+    def viewitems(self) -> ItemsView: ...
+
+OMD = OrderedMultiDict
```

Comment:
Why do we need it here? it is not used anywhere

Reply:
I think these aliases exist in the module? Doesn't seem like they'll do any harm...